igniter/update_window.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+"""Progress window to show when OpenPype is updating/installing locally."""
+import os
+from pathlib import Path
+from .update_thread import UpdateThread
+from Qt import QtCore, QtGui, QtWidgets  # noqa
+from .bootstrap_repos import OpenPypeVersion
+
+
+def load_stylesheet(path: str = None) -> str:
```

Comment:
2 Notes:
- don't give an option to pass path as it doesn't make sense (it would be easier to do `Path(path).read_text()`)
- use `os.path.abspath(__file__)` because it may return path relatively to different path then is this file

Reply:
3rd:
This function and `NiceProgressBar` are duplicated. Maybe they should be in different file from which are used here and in `update_window.py`.